test/index.ts

```diff
@@ -490,13 +490,15 @@ describe('ExtractGQL', () => {
         query {
           author {
             firstName
+            middleName
             lastName
           }
         }`;
       const transformedQuery = gql`
         query {
           author {
             firstName
+            middleName
```

Comment:
This change was necessary because `print` from `graphql` does some sort of caching/memoization and as a result the tests that ran after these tests had `__typename` included in the result of `print()` even when it wasn't supposed to.  I'll leave a comment here explaining the situation.